File: Eshop.Domain/Model/Product.cs
using Eshop.Domain.Relationships;
using Eshop.Domain.ValueObjects;
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace Eshop.Domain.Model
{
    public class Product : BaseEntity
    {
        public string? Name { get; set; }

        public string? Description { get; set; }

        public Money? Price { get; set; }

        public string? Manufacturer { get; set; }

        public bool Discontinued { get; set; }

        // Relationships

        [JsonIgnore]
        public long CategoryId { get; set; }

        public Category? Category { get; set; }

        [JsonIgnore]
        public virtual ICollection<Tag> Tags { get; set; } = new Collection<Tag>();

        [JsonIgnore]
        public virtual IEnumerable<ProductInShoppingCart>? ProductsInShoppingCart { get; set; }

        [JsonIgnore]
        public virtual ICollection<ProductInStore> ProductsInStore { get; set; } = new Collection<ProductInStore>();

        [JsonIgnore]
        public virtual IEnumerable<ProductInOrder>? ProductsInOrder { get; set; }

        public virtual IList<ProductImages> Images { get; set; } = new List<ProductImages>();
    }
}

File: Scraper/product.py
import json

class Product:
    def __init__(self, name, category, price, url, realId, imageBase64, manufacturer, description):
        self.name = name
        self.category = category
        self.price = price
        self.url = url
        self.realId = realId
        self.imageBase64 = imageBase64,
        self.manufacturer = manufacturer
        self.description = description

    def getData(self):
        # return json.dumps(self, default=lambda o: o.__dict__, sort_keys=True, indent=4)
        to_return = {
            "name": self.name,
            "category": self.category,
            "price": self.price,
            "url": self.url,
            "realId": self.realId,
            "manufacturer": self.manufacturer,
            "description": self.description,
        }
        images = []
        print("type", type(self.imageBase64[0]))
        print("len", len(self.imageBase64[0]))
        for image in self.imageBase64[0]:
            print(type(image))
            images.append(image.decode("utf-8"))
        to_return['imageBase64'] = images 

        return to_return